tests/rules/no-unsafe-takeuntil.ts

```diff
@@ -0,0 +1,228 @@
+        r = a.pipe(takeUntil(b), last()).subscribe();
+        r = a.pipe(takeUntil(b), max()).subscribe();
+        r = a.pipe(takeUntil(b), min()).subscribe();
+        r = a.pipe(takeUntil(b), publish()).subscribe();
+        r = a.pipe(takeUntil(b), publishBehavior("x")).subscribe();
+        r = a.pipe(takeUntil(b), publishLast()).subscribe();
+        r = a.pipe(takeUntil(b), publishReplay(1)).subscribe();
+        r = a.pipe(takeUntil(b), reduce((acc, value) => acc + value, "")).subscribe();
+        r = a.pipe(takeUntil(b), share()).subscribe();
+        r = a.pipe(takeUntil(b), shareReplay(1)).subscribe();
+        r = a.pipe(takeUntil(b), skipLast(1)).subscribe();
+        r = a.pipe(takeUntil(b), takeLast(1)).subscribe();
+        r = a.pipe(takeUntil(b), throwIfEmpty()).subscribe();
+        r = a.pipe(takeUntil(b), toArray()).subscribe();
+      `
+    },
+    {
+      code: stripIndent`
+        import { combineLatest, of } from "rxjs";
+        import { takeUntil, tap } from "rxjs/operators";
+
+        const a = of("a");
+        const b = of("b");
+        const c = of("c");
+        const d = of("d");
+
+        const e = a.pipe(takeUntil(d), tap(value => console.log(value))).subscribe();
+      `,
+      options: [
+        {
+          allow: ["tap"]
+        }
+      ]
+    }
+  ],
+  invalid: [
+    {
+      code: stripIndent`
+        import { of } from "rxjs";
+        import { switchMap, takeUntil } from "rxjs/operators";
+
+        const a = of("a");
+        const b = of("b");
+        const c = of("d");
+
+        const d = a.pipe(takeUntil(c), switchMap(_ => b)).subscribe();
+      `,
+      errors: [
+        {
+          messageId: "forbidden",
+          line: 8,
+          column: 18,
+          endLine: 8,
+          endColumn: 27
+        }
+      ]
+    },
+    {
+      code: stripIndent`
+        import { combineLatest, of } from "rxjs";
+        import { takeUntil } from "rxjs/operators";
+
+        const a = of("a");
+        const b = of("b");
+        const c = of("c");
+        const d = of("d");
+
+        const e = a.pipe(takeUntil(d), s => combineLatest(s, b, c)).subscribe();
+      `,
+      errors: [
+        {
+          messageId: "forbidden",
+          line: 9,
+          column: 18,
+          endLine: 9,
+          endColumn: 27
+        }
+      ]
+    },
+    {
+      code: stripIndent`
+        import { combineLatest, of } from "rxjs";
+        import { takeUntil, tap, switchMap } from "rxjs/operators";
+
+        const a = of("a");
+        const b = of("b");
+        const c = of("c");
+
+        const d = a.pipe(takeUntil(c), tap(value => console.log(value)), switchMap(_ => b)).subscribe();
+      `,
+      options: [
+        {
+          allow: ["tap"]
+        }
+      ],
+      errors: [
+        {
+          messageId: "forbidden",
+          line: 8,
+          column: 18,
+          endLine: 8,
+          endColumn: 27
+        }
+      ]
+    },
+    {
+      code: stripIndent`
```

Comment:
Ported over from https://github.com/cartant/rxjs-tslint-rules/blob/master/test/v6/fixtures/issues/49/fixture.ts.lint.

Do you want to have these issues grouped as before?

Reply:
I guess that keeping some reference to the original issue might be useful context, but other than that, I don't mind how they are grouped.